docs/docsite/rst/installation_guide/intro_installation.rst

```diff
@@ -43,71 +45,26 @@ You can find details about control and managed node requirements for each Ansibl
 Selecting an Ansible package and version to install
 ====================================================
 
-Ansible's community packages are distributed in two ways: a minimalist language and runtime package called ``ansible-core``, and a much larger "batteries included" package called ``ansible``, which adds a community-curated selection of :ref:`Ansible Collections <collections>` for automating a wide variety of devices. Choose the package that fits your needs; The following instructions  use ``ansible``, but  you can substitute ``ansible-core``  if you prefer to start with a more minimal package and separately install only the Ansible Collections you require. The ``ansible`` or ``ansible-core`` packages may be available in your operating systems package manager, and you are free to install these packages with your preferred method. These installation instructions only cover the officially supported means of installing the python package with ``pip``.
-
-See the :ref:`Ansible package release status table<ansible_changelogs>` for the ``ansible-core`` version included in the package.
-
-Installing and upgrading Ansible with pipx
-==========================================
-
-On some systems, it may not be possible to install Ansible with ``pip``, due to decisions made by the operating system. In such cases, ``pipx`` is a widely available alternative.
-
-These instructions will not go over the steps to install ``pipx``; If those instructions are needed please continue to the `pipx installation instructions`_ for more information.
-
-.. _pipx installation instructions: https://pypa.github.io/pipx/installation/
-
-.. _pipx_install:
-
-Installing Ansible
-------------------
-
-Use ``pipx`` in your environment to install the Ansible package of your choice for the current user:
-
-.. code-block:: console
-
-    $ pipx install --include-deps ansible
-
-If you prefer to install only the minimal ``ansible-core`` package, run:
+Ansible's community packages are distributed in two ways:
 
-.. code-block:: console
-
-    $ pipx install ansible-core
+* ``ansible-core``: a minimalist language and runtime package containing a set of `builtin modules and plugins <https://docs.ansible.com/ansible/latest/collections/ansible/builtin/index.html>`_.
```

Comment:
This should use a :ref:  I think :ref:`plugins_in_ansible.builtin` should work for both core and package docs.